src/library-authoring/LibraryAuthoringPage.tsx

```diff
@@ -212,6 +212,11 @@ const LibraryAuthoringPage = ({ returnToLibrarySelection }: LibraryAuthoringPage
     />
   ) : undefined;
 
+  const extraFilter = [`context_key = "${libraryId}"`];
+  if (componentPickerMode) {
+    extraFilter.push('last_published IS NOT NULL');
```

Comment:
Keep in mind, that when we're using the component picker to add things to collections like https://github.com/openedx/frontend-app-authoring/issues/1173 which @rpenido is working on, we want to show draft components (as before / as normal). It's only when using the component picker to add to a course that we want this published filter.

Reply:
I updated the code to use query params to choose 'draft' or 'published'

Reply:
Perfect!